src/raftstore/coprocessor/config.rs

```diff
@@ -27,10 +27,12 @@ pub struct Config {
     /// be region_split_size (or a little bit smaller).
     pub region_max_size: ReadableSize,
     pub region_split_size: ReadableSize,
+    pub half_split_bucket_size: ReadableSize,
```

Comment:
what is this config used for?

Reply:
It's used for computing region's mid_key(split_key), this value has a direct influence on split_key